news/2023/2023-10-23-beatmap-spotlights-season-10-autumn-2023.md

```diff
@@ -0,0 +1,201 @@
+---
+layout: post
+title: "Beatmap Spotlights Season 10: Autumn 2023"
+date: 2023-10-23 18:00:00 +0000
+---
+
+You heard it right, the Beatmap Spotlights are back with its **10th Season**! With how moody autumn days can get, you can fix that by competing on the best beatmapping creations of last months right now!
+
+![](/wiki/shared/news/banners/beatmap-spotlights.jpg)
+
+After a bit of a breathing break to sort out some of the management challenges surrounding Beatmap Spotlights, we'd like to announce that we are back with the **10th season of Beatmap Spotlights**. With Autumn 2023 season starting, we'd like to thank you for staying with us for 2-digit amount of iterations over the years of playlist-oriented history of Beatmap Spotlights! Even though this season will remain rather regular, we are preparing some bigger-scheme changes for the next ones, so stay prepared for that!
+
+If you just can't wait to try everything out, **the Beatmap Spotlights lobbies are now open for play NOW on osu!(lazer)**, so go get involved and start setting scores.
+
+If you still don't have any idea about what the entire Beatmap Spotlights thing could be, just read on below to find out how to jump into the world of playlist-oriented competition now:
+
+## How do I participate in the Beatmap Spotlights?
+
+To participate in the Beatmap Spotlights, go grab yourself the latest version of the osu!(lazer) client from this [download page](https://osu.ppy.sh/home/download).
+
+Once you're done installing and logging into your account, head on over to the playlists tab in the play section and find the Spotlights lobbies as denoted by a little pink bar. You can also toggle the filter to show only the Spotlights playlists in the top-right corner. 
```

Comment:
might be a lot of quotes but makes it more understandable for what they are supposed to be exactly finding. At least this is what we usually do in the help forum.

```Suggestion
Once you're done installing and logging into your osu! account, head on over to the "playlists" tab in the "play" section and find the "Spotlights" lobbies, denoted by a little pink bar. You can also toggle the filter to show only the "Spotlights" playlists in the top-right corner.
```

Reply:
wiki convention would be to do something like this, with interface text in code blocks

```suggestion
Once you're done installing and logging into your account, head on over to the `playlists` tab in the `play` section and find the Spotlights lobbies, denoted by a little pink bar. You can also toggle the filter to show only the Spotlights playlists in the top-right corner. 
```